design-tokens/tokens/layer1/base.json

```diff
@@ -116,6 +116,48 @@
           "value": "25rem",
           "type": "dimension"
         }
+      },
+      "height": {
+        "100": {
+          "value": "0.25rem",
+          "type": "dimension"
+        },
+        "200": {
+          "value": "0.5rem",
+          "type": "dimension"
+        },
```

Comment:
We're missing some values:
RadioButtonGroup's "SelectionIndicator" is using `echoes-dimension-size-75` for its height (so we're missing 150)

Reply:
We have a specific layer for "Border Widths" outside of the "Sizes" layer

Reply:
Ok, I've just added the 150 value

Reply:
Then we're missing the right border width for the Radiobutton in the Selects, we only have the default 1px currently